scripts/variantstore/gvs-overview.md

```diff
@@ -0,0 +1,165 @@
+# Genomic Variant Store pipeline overview
+ 
+| Pipeline Version | Date Updated | Documentation Authors | Questions or Feedback |
+| :----: | :---: | :----: | :--------------: |
+| [GvsJointVariantCalling](https://github.com/broadinstitute/gatk/blob/rc-vs-483-beta-user-wdl/scripts/variantstore/wdl/GvsJointVariantCalling.wdl) | June, 2022 | [Kaylee Mathews](mailto:kmathews@broadinstitute.org) and [Aurora Cremer](mailto:aurora@broadinstitute.org) | If you have questions or feedback, contact the [Broad Variants team](mailto:variants@broadinstitute.org) |
```

Comment:
change email to warp-pipelines-help@broadinstitute.org

Reply:
@kbergin suggested we use the Variants email. Is that still what you'd like to do Kylee or would you rather use the WARP help email?